aws/resource_aws_opsworks_stack.go

```diff
@@ -366,6 +367,16 @@ func resourceAwsOpsworksStackRead(d *schema.ResourceData, meta interface{}) erro
 	}
 	resourceAwsOpsworksSetStackCustomCookbooksSource(d, stack.CustomCookbooksSource)
 
+	tags, err := keyvaluetags.OpsworksListTags(client, arn)
```

Comment:
We weren't getting tags in the resource's `Read` method previously.

Reply:
Overall this is a good change as it helps Terraform perform drift detection for tags (and fixes their import) when the Opsworks Stack is in the same region as the provider, although I worry a little bit about the cross-region abilities of this resource. Going forward tag listing errors for invalid ARNs will now always report. See also: #6381

I'm okay merging this in an effort of moving the resource into a better spot, although this may mean other bug reports similar to #6381 will start coming if the reads start failing now.